src/main/java/sg/ncl/MainController.java

```diff
@@ -179,6 +179,12 @@ public String jointeam() {
         return "jointeam";
     }
 
+
+    @RequestMapping("/Error_OpenStack")
+    public String Error_OpenStack() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00100)